docs/customer-swagger.yaml

```
@@ -608,6 +608,70 @@ paths:
               examples:
                 InternalServerErrorExample:
                   $ref: '#/components/examples/InternalServerErrorProblemExample'
+  /{customerIdentifier}/channel-claim/{channelClaimIdentifier}:
+    delete:
+      summary: Unclaim a channel claim
+      operationId: UnclaimChannelClaim
+      security:
+        - CognitoUserPool: [ "https://api.nva.unit.no/scopes/frontend", "aws.cognito.signin.user.admin" ]
+      parameters:
+        - name: customerIdentifier
+          description: UUID identifier of the customer to unclaim channel claim for
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/CustomerIdentifier'
+        - name: channelClaimIdentifier
+          description: UUID identifier of the channel claim to unclaim
```

Not sure, but might read better? 
```suggestion
          description: UUID identifier of the channel to unclaim
```

---

Agree